server/index.js

```diff
@@ -1,18 +1,20 @@
-const express = require('express')
-// var bodyParser = require('body-parser')
-const app = express()
+var express = require('express')
+var app = express()
 // app.use(bodyParser.json())
-const port = 3000;
-const mongoose = require('mongoose');
-// var faker = require('faker');
-// mongoose.connect('mongodb://localhost/test', {useNewUrlParser: true});
-// const DB = require('./database');
+var port = 3007;
+
+var DB = require('/Users/sinamb/Documents/Programs/SinaService/database/index.js');
 
 app.use(express.static('/Users/sinamb/Documents/Programs/SinaService/client/dist'))
 
 
-app.get('/updates', (req, res) => {
-  //run a function that creates 100 random objects
+app.get('/updates/:itemId', (req, res) => {
+  var itemId = req.params.itemId;
+  DB.retrieveUpdates(itemId)
+    .then( (updateInfo) => {
+      console.log(updateInfo);
+      res.send(updateInfo);
+    })
 
```

Comment:
You need a .catch if your retrieveUpdates function fails so you can send back an appropriate error to the client